docs/source/API/algorithms/std-algorithms/StdNonModSeq.md

```diff
@@ -151,16 +150,999 @@ struct EqualsValue
+auto find_end(const ExecutionSpace& ex,
+              const ::Kokkos::View<DataType1, Properties1...>& view,                  (3)
+              const ::Kokkos::View<DataType2, Properties2...>& s_view);
+
+template <class ExecutionSpace, class DataType1, class... Properties1,
+          class DataType2, class... Properties2>
+auto find_end(const std::string& label, const ExecutionSpace& ex,
+              const ::Kokkos::View<DataType1, Properties1...>& view,                  (4)
+              const ::Kokkos::View<DataType2, Properties2...>& s_view);
+
+// overload set 2: binary predicate passed
+template <class ExecutionSpace, class IteratorType1, class IteratorType2,
+          class BinaryPredicateType>
+IteratorType1 find_end(const ExecutionSpace& ex, IteratorType1 first,
+                       IteratorType1 last, IteratorType2 s_first,                     (5)
+                       IteratorType2 s_last, const BinaryPredicateType& pred);
+
+template <class ExecutionSpace, class IteratorType1, class IteratorType2,
+          class BinaryPredicateType>
+IteratorType1 find_end(const std::string& label, const ExecutionSpace& ex,
+                       IteratorType1 first, IteratorType1 last,                       (6)
+                       IteratorType2 s_first, IteratorType2 s_last,
+                       const BinaryPredicateType& pred);
+
+template <class ExecutionSpace, class DataType1, class... Properties1,
+          class DataType2, class... Properties2, class BinaryPredicateType>
+auto find_end(const ExecutionSpace& ex,
+              const ::Kokkos::View<DataType1, Properties1...>& view,                  (7)
+              const ::Kokkos::View<DataType2, Properties2...>& s_view,
+              const BinaryPredicateType& pred);
+
+template <class ExecutionSpace, class DataType1, class... Properties1,
+          class DataType2, class... Properties2, class BinaryPredicateType>
+auto find_end(const std::string& label, const ExecutionSpace& ex,
+              const ::Kokkos::View<DataType1, Properties1...>& view,                  (8)
+              const ::Kokkos::View<DataType2, Properties2...>& s_view,
+              const BinaryPredicateType& pred);
+```
+
+### Description
+Searches for the last occurrence of the sequence [s_first, s_last) in the range [first, last) in (1,2).
+Searches for the last occurrence of the `s_view` in `view`  in (3,4).
+Searches for the last occurrence of the sequence [s_first, s_last) in the range [first, last) in (5,6) for which `pred` evaluates to true.
+Searches for the last occurrence of the `s_view` in `view`  in (7,8) for which `pred` evaluates to true.
+
+### Parameters and Requirements
+- `execspace`, `first`, `last`, `view` and `count` similar to [#Kokkos::Expeimental::for_each_n](#kokkosexpeimentalfor_each_n).
+- `label`:
+    - 1,5: The default string is "Kokkos::search_n_iterator_api_default".
+    - 3,7: The default string is "Kokkos::search_n_view_api_default".
```

Comment:
wrong string for label

Reply:
Thanks for pointing it out. I made the same mistake in a few other places. Fixed it now.